csrankings-d.csv

```diff
@@ -844,6 +845,7 @@ Deepak Garg 0001 [SWS],Max Planck Society,http://www.mpi-sws.org/~dg,BCXxazwAAAA
 Deepak K. Ghodgaonkar,DAIICT,http://www.daiict.ac.in/daiict/people/faculty.html,NOSCHOLARPAGE
 Deepak Kapur,University of New Mexico,https://www.cs.unm.edu/~kapur,U6XKBVIAAAAJ
 Deepak Khemani,IIT Madras,http://aidblab.cse.iitm.ac.in/khemani,NOSCHOLARPAGE
+Deepak Mishra,IIT Jodhpur,https://sites.google.com/iitj.ac.in/dmishra,-rOCu6sAAAAJ

```

Comment:
Missing disambiguation suffix - see https://dblp.org/pid/65/6758.html. Check all names against DBLP, per instructions.

Reply:
Please address this issue ASAP; I am closing old PRs by the end of this week.